docs/usage/applications.rst

```diff
@@ -84,152 +85,139 @@ During shutdown, they are executed in the following order:
+The :paramref:`~litestar.app.Litestar.after_exception` hook takes a
+:class:`sync or async callable <litestar.types.AfterExceptionHookHandler>` that is called with two arguments:
+the ``exception`` that occurred and the ASGI ``scope`` of the request or websocket connection.
 
 .. literalinclude:: /examples/application_hooks/after_exception_hook.py
     :caption: After Exception Hook
-    :language: python
-
 
-.. attention::
-
-    This hook is not meant to handle exceptions - it just receives them to allow for side effects.
+.. attention:: This hook is not meant to handle exceptions - it just receives them to allow for side effects.
     To handle exceptions you should define :ref:`exception handlers <usage/exceptions:exception handling>`.
 
 Before Send
 ^^^^^^^^^^^
 
-The ``before_send`` hook takes a :class:`sync or async callable <litestar.types.BeforeMessageSendHookHandler>` that is called when
-an ASGI message is sent. The hook receives the message instance and the ASGI ``scope``.
+The :paramref:`~litestar.app.Litestar.before_send` hook takes a
+:class:`sync or async callable <litestar.types.BeforeMessageSendHookHandler>` that is called when an ASGI message is
+sent. The hook receives the message instance and the ASGI ``scope``.
 
 .. literalinclude:: /examples/application_hooks/before_send_hook.py
     :caption: Before Send Hook
-    :language: python
-
 
 Initialization
 ^^^^^^^^^^^^^^
 
-Litestar includes a hook for intercepting the arguments passed to the :class:`Litestar constructor <litestar.app.Litestar>`,
-before they are used to instantiate the application.
+Litestar includes a hook for intercepting the arguments passed to the
+:class:`Litestar constructor <litestar.app.Litestar>`, before they are used to instantiate the application.
 
-Handlers can be passed to the ``on_app_init`` parameter on construction of the application, and in turn, each will receive
-an instance of :class:`AppConfig <litestar.config.app.AppConfig>` and must return an instance of same.
+Handlers can be passed to the :paramref:`~.app.Litestar.on_app_init` parameter on construction of the application,
+and in turn, each will receive an instance of :class:`~.config.app.AppConfig` and must return an instance of same.
 
 This hook is useful for applying common configuration between applications, and for use by developers who may wish to
 develop third-party application configuration systems.
 
-.. note::
-
-    ``on_app_init`` handlers cannot be ``async def`` functions, as they are called within ``Litestar.__init__()``,
-    outside of an async context.
+.. note:: :paramref:`~.app.Litestar.on_app_init` handlers cannot be :ref:`python:async def` functions, as they are
+    called within :paramref:`~litestar.app.Litestar.__init__`, outside of an async context.
 
 .. literalinclude:: /examples/application_hooks/on_app_init.py
     :caption: After Exception Hook
```

Comment:
Incorrect caption

Reply:
fixed in 689d47537bb368d6a660a6673647f00b92528ddf